subprojects/platform-play/src/testFixtures/groovy/org/gradle/play/integtest/fixtures/PlayMultiVersionIntegrationTest.groovy

```diff
@@ -19,7 +19,23 @@ package org.gradle.play.integtest.fixtures
 import org.gradle.api.JavaVersion
 import org.gradle.integtests.fixtures.MultiVersionIntegrationSpec
 import org.gradle.integtests.fixtures.TargetCoverage
+import org.gradle.integtests.fixtures.executer.ExecutionFailure
+import org.gradle.integtests.fixtures.executer.ExecutionResult
 
 @TargetCoverage({ JavaVersion.current().isJava8Compatible() ? PlayCoverage.ALL : PlayCoverage.PLAY23_OR_EARLIER })
 abstract class PlayMultiVersionIntegrationTest extends MultiVersionIntegrationSpec {
+
+    protected ExecutionFailure fails(String... tasks) {
+        if (version.toString().startsWith('2.2')) {
```

Comment:
We are repeating this expression 4-5 times in this change. Let's externalize it into a fixture method.

Reply:
Done.